OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/NotificationControllerTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using OutOfSchool.WebApi.Controllers.V1;
+using OutOfSchool.WebApi.Services;
+
+namespace OutOfSchool.WebApi.Tests.Controllers;
+
+[TestFixture]
+public class NotificationControllerTests
+{
+    private NotificationController controller;
+    private Mock<INotificationService> serviceMock;
+    private Mock<HttpContext> httpContext;
+    private string userId;
```

Comment:
(Redundant fields) Only `controller` is used in tests. Other fields are scoped to `SetUp` and could be moved to local variables there or even inlined

Reply:
Fixed. Thank you!